src/main/java/edu/ksu/canvas/requestOptions/UpdateCoursesOptions.java

```diff
@@ -0,0 +1,43 @@
+package edu.ksu.canvas.requestOptions;
+
+import java.util.List;
+
+/**
+ * Encapsulates the allowed input to the Canvas API UPDATE call for courses.
+ */
+public class UpdateCoursesOptions extends BaseOptions {
+
+    /**
+     * 	The action to take on each course
+     */
+    public enum EventType {
+
+        OFFER, CONCLUDE, DELETE, UNDELETE;
+
+        @Override
+        public String toString() {
+            return name().toLowerCase();
+        }
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param eventType whether to delete or conclude the course
+     */
+    public UpdateCoursesOptions(EventType eventType) {
+        addSingleItem("event", eventType.toString());
+    }
+
+    /**
+     * List of ids of courses to update. At most 500 courses may be updated in one call.
+     *
+     * @param courseIds IDs to update
+     * @return This object to allow adding more options
+     */
+    public UpdateCoursesOptions courseIds(List<Integer> courseIds) {
+        addNumberList("course_ids[]", courseIds);
+        return this;
+    }
```

Comment:
Normally I would expect this to accept `String` rather than a number so that you can use SIS IDs as well as Canvas course IDs (https://canvas.instructure.com/doc/api/file.object_ids.html).
If it doesn't support SIS ID expansion then it should at least be `Long` as all numbers in the Canvas API (for IDs) are 64 bit.